templates/careers/hiring-process/index.html

```diff
@@ -16,29 +18,30 @@
 
 {% block content %}
 
-  <section class="p-section--hero">
-    <div class="row--50-50">
+  <section class="p-section--hero p-section--shallow">
```

Comment:
The hero section gets canceled out by the shallow so we can remove

@mattea-turic let me know if we should open a vanilla ticket for this as both the hero section and the jinja hero pattern share the original bottom padding

```suggestion
  <section class="p-section--shallow">
```

Reply:
I did this because the hero section has a top padding which is not there in the shallow section. So this way I get the top padding of the hero with the bottom padding of the shallow

Reply:
@mattea-turic @mtruj013 ^ waiting on what to do

Reply:
@abhigyanghosh30 I've not seen us use this combination before but there doesn't seem to be any documentation advising against it. I've already approved the pr, so please feel free to merge as is